kernel/comps/logger/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! Logger System
+//!
+//! Users need to specify one and only one logger feature for this crate to enable
```

Comment:
I don't understand what "specify one and only one logger feature for this crate" wants to mean.